### PR TITLE
modified to work with brew installed modules on mac osx

### PR DESCRIPTION
**Pull request by csayres**

### CODE REVIEW

**Review by joelbrownstein — COMMENTED** [review]
Hi Conor, the path change you made at line 67 would have broken other versions of modules (e.g. the one at Utah), so I changed your fix so that it tries both paths in series, which should make it work both at Utah and your brew version.  Can you do a git pull and see if its OK?

File: python/sdss_install/utils/module.py [hunk 1]
```diff
@@ -64,7 +64,11 @@ def set_modules_home(self):
                 self.modules_home = {'dir': modules_home}
                 self.modules_home['lmod'] = join(modules_home, "libexec", "lmod")
                 self.modules_home['lua'] = join(modules_home, "init", "lmodrc.lua")
-                self.modules_home['tcl'] = join(modules_home, "libexec", "modulecmd.tcl")
+                tcl = []
+                tcl.append(join(modules_home, "modulecmd.tcl"))
+                tcl.append(join(modules_home, "libexec", "modulecmd.tcl"))
+                for self.modules_home['tcl'] in tcl:
+                    if exists(tcl): break
```
Comment (nbmurphy) [inline python/sdss_install/utils/module.py:71]: If neither `join(modules_home, "modulecmd.tcl")` nor `join(modules_home, "libexec", "modulecmd.tcl")` exist then `self.modules_home['tcl']` will be set to the latter. Is this the desired behavior? If so would the following be more readable?
```
tcl = join(modules_home, "modulecmd.tcl")
libexec_tcl = join(modules_home, "libexec", "modulecmd.tcl")
self.modules_home['tcl'] = tcl if exists(tcl) else  libexec_tcl
```
Did you instead intend to also append `None` at the end of tcl, so if neither exist then `self.modules_home['tcl'] = None`?

File: python/sdss_install/utils/module.py [hunk 2]
```diff
@@ -79,8 +83,8 @@ def set_modules_lang(self):
 
     def set_tclsh(self):
         '''Set tclsh exec directory.'''
-        self.tclsh = str()
-        if self.ready:
+        self.tclsh = None
+        if self.ready and self.modules_lang['tcl']:
```
Comment (nbmurphy) [inline python/sdss_install/utils/module.py:87]: In the method `set_modules_home()`, it seems that `self.modules_lang['tcl']` is always set to something non-`False`. So does the `and self.modules_lang['tcl']` do anything in `if self.ready and self.modules_lang['tcl']:`?

**Comment by joelbrownstein** [comment]
this pull request was replaced with cs2 branch

**Pull request closed without merging**